code/game/objects/items/robot/robot_items.dm

```diff
@@ -70,7 +70,9 @@
 	switch(mode)
 		if(0)
 			if(M.health >= 0)
-				if(user.zone_selected == BODY_ZONE_HEAD)
+				if(isanimal(M))
+					M.attack_hand(user)
+				else if(user.zone_selected == BODY_ZONE_HEAD)
```

Comment:
i think you want to early return and just have `if` instead of `else if`.

this proc lmao...

Reply:
Yeah I uh... wanted to spend as little time as humanly possible looking at that code.

I'll try that and see if it messes anything up.

Reply:
It seems to work.